### PR TITLE
Adding eip155-2139927552 - SKALE - Exorde Network 

### PR DESCRIPTION
**Pull request by MathiasExorde**
"url": "https://exorde.network/"
"faucetUrl": "http://faucet.exorde.network/"
"description": "The Web3 protocol collecting data from the Web jungle to capture topics, news, reputation, and trends worldwide in real-time."

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/icons/exorde.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafybeidq7g4nbgkok7m3syvqhabxivrpj5ryreeabye3mlqdw64cqmsvam",
```
Comment (ligi) [inline _data/icons/exorde.json:3]: This is a directory and not a file - so the contentHash in IconDownload is wrong also
```
identify bafybeidq7g4nbgkok7m3syvqhabxivrpj5ryreeabye3mlqdw64cqmsvam/exorde-logo.png 
bafybeidq7g4nbgkok7m3syvqhabxivrpj5ryreeabye3mlqdw64cqmsvam/exorde-logo.png PNG 290x401 290x401+0+0 8-bit sRGB 3114B 0.000u 0:00.000

```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by MathiasExorde — COMMENTED** [review]
Changed folder IPFS hash to file hash

**Comment by MathiasExorde** [comment]
is it all good now? @ligi

File: _data/chains/eip155-2139927552.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://exorde.network/",
+  "shortName": "exorde-network",
+  "chainId": 2139927552,
+  "networkId": 2139927552,
+  "explorers": [{
+    "name": "blockscout",
+    "url": "https://light-vast-diphda.explorer.mainnet.skalenodes.com",
+
```
Comment (ligi) [inline _data/chains/eip155-2139927552.json:22]:
```suggestion
```

File: _data/chains/eip155-2139927552.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Exorde Network",
+  "chain": "EXORDE",
+  "network": "mainnet",
```
Comment (ligi) [inline _data/chains/eip155-2139927552.json:4]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

File: _data/chains/eip155-2139927552.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "symbol": "sFUEL",
+    "decimals": 18
+  },
+  "infoURL": "https://exorde.network/",
+  "shortName": "exorde-network",
+  "chainId": 2139927552,
+  "networkId": 2139927552,
+  "explorers": [{
+    "name": "blockscout",
+    "url": "https://light-vast-diphda.explorer.mainnet.skalenodes.com/",
```
[the remark this answers is not in the document]
Comment (robledo6) [reply _data/chains/eip155-2139927552.json]: No sé hacer esto sheen

**Pull request closed without merging**